std/net/curl.d

```diff
@@ -1510,12 +1511,8 @@ private mixin template WorkerThreadProtocol(Unit, alias units)
     }
 }
 
-// Workaround bug #2458
```

Comment:
the bug is fixed, but afaik having struct outsides is currently better because it avoids the long names. Ideally we would disallow access to it, but that would be yet another deprecation cycle?